groupAnagrams.js

```diff
@@ -0,0 +1,44 @@
+// This file contains solution to the following problem:
+// https://leetcode.com/problems/group-anagrams/
+
+const groupAnagrams = (strs) => {
+  let strsArray = strs;
+  const result = [];
+  while (strsArray.length > 0) {
+    const grouppedAnagrams = [strsArray[0]];
+    for (let j = 1; j < strsArray.length; j += 1) {
+      if (strsArray[0].length !== strsArray[j].length) {
```

Comment:
use `grouppedAnagrams ` instead of `strsArray[0]`

Reply:
const grouppedAnagrams is different from strsArray[0], because I dynamically change strsArray.